### PR TITLE
Add CLI utils tests

### PR DESCRIPTION
**Pull request by strickvl**
Tests for the CLI utils where it makes sense to test. (I.e. this ignores all the functions that exist purely to print some formatted version of text using `rich`).

## Pre-requisites
Please ensure you have done the following:
- [ ] I have read the **CONTRIBUTING.md** document.
- [ ] If my change requires a change to docs, I have updated the documentation accordingly.
- [ ] If I have added an integration, I have updated the [integrations](https://docs.zenml.io/component-gallery/integrations) table and the [corresponding website section](https://zenml.io/integrations).
- [ ] I have added tests to cover my changes.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Other (add details above)

### CODE REVIEW

File: tests/harness/cfg/deployments_dev.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# A list of ZenML deployments, either local or remote (external), that
```
Comment (stefannica) [inline tests/harness/cfg/deployments_dev.yaml:1]: This file slipped through, shouldn't be here.

File: tests/harness/cfg/deployments_dev.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+#
+deployments:
+
+  - name: devel
+    description: >-
+      Development deployment.
+    type: server
+    setup: external
+    config:
+      url: sqlite:////Users/strickvl/Library/Application Support/zenml/local_stores/default_zen_store/zenml.db
```
Comment (safoinme) [inline tests/harness/cfg/deployments_dev.yaml:66]: Are we sure we want to have this URL here, it seems like very specific to your env

**Review by stefannica — CHANGES_REQUESTED** [review]
One extra file, but LGTM otherwise.

File: tests/integration/functional/cli/test_utils.py [hunk 3]
```diff
@@ -26,10 +28,75 @@ def test_temporarily_setting_the_active_stack():
...
+
+    # test that the file contents are returned
+    assert (
+        cli_utils.expand_argument_value_from_file(
+            name="sample_text", value=f"@{file_path}"
+        )
+        == sample_text_value
+    )
+
+    assert (
```
Comment (safoinme) [inline tests/integration/functional/cli/test_utils.py:61]: do we ever want to test a non-existing file?

File: tests/harness/cfg/deployments_dev.yaml (hunk 2 above)
Comment (strickvl) [reply tests/harness/cfg/deployments_dev.yaml:66]: Oh. no idea why this file made it in here... I'll remove...

File: tests/integration/functional/cli/test_utils.py (hunk 3 above)
Comment (strickvl) [reply tests/integration/functional/cli/test_utils.py:61]: Tested now.

File: tests/integration/functional/cli/test_utils.py [hunk 4]
```diff
@@ -26,10 +28,75 @@ def test_temporarily_setting_the_active_stack():
...
+        cli_utils.expand_argument_value_from_file(
+            name="text_not_from_file", value=not_from_file_value
+        )
+        == not_from_file_value
+    )
+
+
+def test_parsing_name_and_arguments():
```
[the remark this answers is not in the document]
Comment (strickvl) [reply tests/integration/functional/cli/test_utils.py:75]: Added a test for this, but I think not...

**Review by safoinme — APPROVED** [review]
LGTM!

File: tests/integration/functional/cli/test_utils.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (safoinme) [reply tests/integration/functional/cli/test_utils.py:75]: We should probably do on some, but I guess we never saw an error from it, so it's not time for it.

**Review by stefannica — APPROVED** [review]

**Pull request merged**